src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventProvider.cs

```diff
@@ -12,7 +12,7 @@
 using System.Globalization;
 using System.Numerics;
 using System.Runtime.InteropServices;
-#if (CORECLR || MONO) && TARGET_WINDOWS
+#if (CORECLR || MONO || CORERT) && TARGET_WINDOWS
```

Comment:
```suggestion
#if !ES_BUILD_STANDALONE && TARGET_WINDOWS
```

Reply:
How do we want to approach these cleanups? Do we want to accumulate them in the branch, or do the cleanup in the runtime repo?

Reply:
I would accumulate one liners like this one in the branch and flush them out once in a while.